projects/blueberry/index.js

```diff
@@ -1,6 +1,33 @@
 const { compoundExports2 } = require('../helper/compound')
+const { abi } = require('./abi.json');
+const sdk = require('@defillama/sdk');
+
+const BANK = '0x9b06eA9Fbc912845DF1302FE1641BEF9639009F7';
+
+async function tvl(timestamp, block, chainBlocks) {
+  let total = 0;
+  const nextPositionId = await sdk.api.abi.call({
+    target: BANK,
+    abi: abi['getNextPositionId'],
+    block
+  }).output;
+
+  for (let i = 0; i < nextPositionId; i++ ) {
+    const positionValue = await sdk.api.abi.call({
+      target: BANK,
+      abi: abi['getPositionValue'],
```

Comment:
hi, what is the logic of position value calculation? does it already subtract debt from the value?

Reply:
Hi the logic of the position value calculation, get's the collateral value and the rewards value added together. The debt value is included in that calculation because your debt is getting deployed into the position.